mod/custom_index/views/default/plugins/custom_index/settings.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/* @var $custom_index \ElggPlugin */
+$custom_index = elgg_extract('entity', $vars);
```

Comment:
we usually keep this as `$entity` or maybe `$plugin`

Reply:
i know, but later we make a `$plugin` so i did want to make clear which plugin i was talking about here